src/types/line.js

```diff
@@ -244,27 +247,17 @@ function drawLabel(ctx, line, chartArea) {
   }
 
   ctx.fillStyle = label.color;
-  if (isArray(label.content)) {
+  if (content instanceof Image) {
+    const x = -(width / 2) + xPadding + borderWidth / 2;
+    const y = -(height / 2) + yPadding + borderWidth / 2;
+    ctx.drawImage(content, x, y, width - (2 * xPadding) - borderWidth, height - (2 * yPadding) - borderWidth);
+  } else {
+    const labels = isArray(content) ? content : [content];
```

Comment:
'middle' works so much better:
```suggestion
    const labels = isArray(content) ? content : [content];
    const x = calculateLabelXAlignment(label, width);
    const y = -(labels.length - 1) * font.lineHeight / 2;
    ctx.textBaseline = 'middle';
    ctx.textAlign = label.textAlign;
    labels.forEach((l, i) => ctx.fillText(l, x, y + (i * font.lineHeight)));
```